src/app/public/modules/datepicker/datepicker.component.ts

```diff
@@ -202,11 +202,18 @@ export class SkyDatepickerComponent implements OnDestroy, OnInit {
     private coreAdapter: SkyCoreAdapterService,
     private overlayService: SkyOverlayService,
     @Optional() public inputBoxHostService?: SkyInputBoxHostService,
-    @Optional() public themeSvc?: SkyThemeService
+    @Optional() themeSvc?: SkyThemeService
   ) {
     const uniqueId = nextId++;
     this.calendarId = `sky-datepicker-calendar-${uniqueId}`;
     this.triggerButtonId = `sky-datepicker-button-${uniqueId}`;
+
+    // Update icons when theme changes.
+    themeSvc?.settingsChange
+    .pipe(takeUntil(this.ngUnsubscribe))
+    .subscribe(() => {
+      this.changeDetector.markForCheck();
+    });
```

Comment:
☝️ Not in the scope of this PR, but upgrading the theme library revealed a bug. Icons weren't updating when the user switches themes back and forth. It was caught in our unit tests, so it needed fixed for this work to continue. This fix ensures we force change detection to fire when the theme switches.